modules/ossm-rn-new-features.adoc

```diff
@@ -19,11 +19,102 @@ Module included in the following assemblies:
 * *Policy Enforcement* - Apply organizational policy to the interaction between services, ensure access policies are enforced and resources are fairly distributed among consumers. Policy changes are made by configuring the mesh, not by changing application code.
 * *Telemetry* - Gain understanding of the dependencies between services and the nature and flow of traffic between them, providing the ability to quickly identify issues.
 
+== New features {SMProductName} 2.2
+
+This release of {SMProductName} adds new features and enhancements, and is supported on OpenShift Container Platform 4.9, and 4.10.
+
+=== Component versions included in {SMProductName} version {SMProductVersion}
+
+|===
+|Component |Version
+
+|Istio
+|1.12.7
+
+|Envoy Proxy
+|1.17.1
+
+|Jaeger
+|1.30.2
+
+|Kiali
+|1.48.0
+|===
+
+=== `WasmPlugin` API
+This release changes the implementation of `WebAssembly` extensions to use the `WasmPlugin` API instead of the `ServiceMeshExtention` API, which has been deprecated.
+
+=== ROSA support
+This release introduces service mesh support for Red Hat OpenShift on AWS (ROSA), including multi-cluster federation.
+
+=== `istio-node` DaemonSet renamed
+This release, the `istio-node` DaemonSet is renamed to `istio-cni-node` to match the name in upstream Istio.
+
+=== Envoy sidecar networking changes
+Istio 1.10 updated Envoy to send traffic to the application container using `eth0` rather than `lo` by default.
+
+=== Service Mesh Control Plane 1.1
+This release marks the end of support for service mesh control planes based on Service Mesh 1.1 for all platforms.
+
+=== Istio 1.12 Support
+
+{SMProductShortName} 2.2 is based on Istio 1.12, which brings in new features and product enhancements. While many Istio 1.12 features are supported, the following exceptions should be noted:
+
+* Telemetry API is a tech preview feature.
+* gRPC Proxyless Service Mesh is a tech preview feature.
+* External control plane is not a supported feature.
+* Gateway injection is not a supported feature.
+* Discovery selectors is not a supported feature.
+* AuthPolicy Dry Run is a tech preview feature.
+
+=== Kubernetes Gateway API
+Kubernetes Gateway API is a technology preview feature that is disabled by default.
+
+To enable the feature, set the following environment variables for the `Istiod` container in `ServiceMeshControlPlane`:
+
+[source,yaml]
+----
+spec:
+  runtime:
+    components:
+      pilot:
+        container:
+          env:
+            PILOT_ENABLE_GATEWAY_API: true
+            PILOT_ENABLE_GATEWAY_API_STATUS: true
+            # and optionally, for the deployment controller
+            PILOT_ENABLE_GATEWAY_API_DEPLOYMENT_CONTROLLER: true
+----
+
+In the Gateway resource, the `listeners.allowedRoutes.namespaces` setting does not work. If you try to use it, then the Gateway resource ignores the setting and always uses the default `sameNamespace` setting instead.
```

Comment:
I'd say that "[restricting route attachment](https://gateway-api.sigs.k8s.io/concepts/api-overview/#restricting-route-attachment) on Gateway API listeners is currently only possible with the `SameNamespace` or `All` settings. Usage of label selectors in `listeners.allowedRoutes.namespaces.from.selectors` will currently be ignored and revert to the default behaviour (sameNamespace)."